examples/call_rest_api.py

```diff
@@ -140,7 +147,41 @@
 orders = paradex.api_client.fetch_orders()
 logger.info(f"ALL {orders=}")
 logger.info("Sleeping for 10 seconds")
-time.sleep(10)
+time.sleep(3)
+
+# Test modify
+order_client_id = f"test_modify_{datetime.now().strftime('%Y%m%d%H%M%S')}"
+initial_order = Order(
+    market="BTC-USD-PERP",
+    order_type=OrderType.Limit,
+    order_side=OrderSide.Buy,
+    size=Decimal("0.01"),
+    limit_price=Decimal(10000),
+    client_id=order_client_id,
+    instruction="POST_ONLY",
+    reduce_only=False,
+)
+response = paradex.api_client.submit_order(order=initial_order)
+logger.info(f"Initial Order {response=}")
+time.sleep(3)
+modify_order_id = response.get("id")
+modify_order = Order(
+    order_id=modify_order_id,
+    market="BTC-USD-PERP",
+    order_type=OrderType.Limit,
+    order_side=OrderSide.Buy,
+    size=Decimal("0.01"),
+    limit_price=Decimal(9500),
+    client_id=buy_client_id,
+    instruction="POST_ONLY",
+    reduce_only=False,
+)
```

Comment:
Could we use the above created orders instead of submitted a new order here? For example `buy_client_id`